src/main/java/qna/domain/Answers.java

```diff
@@ -0,0 +1,77 @@
+package qna.domain;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import qna.CannotDeleteException;
+import qna.domain.message.ExceptionMessage;
+
+@Embeddable
+public class Answers {
+    @OneToMany(mappedBy = "question")
+    private Set<Answer> values;
+
+    protected Answers() {
+        this.values = new HashSet<>();
+    }
+
+    private Answers(Set<Answer> values) {
+        this.values = values;
+    }
+
+    public static Answers init() {
+        return new Answers(new HashSet<>());
+    }
+
+    public static Answers from(List<Answer> answers) {
+        return new Answers(new HashSet<>(answers));
+    }
+
+    public void add(Answer answer) {
+        this.values.add(answer);
+    }
+
+    public void remove(Answer answer) {
+        this.values.remove(answer);
+    }
+
+    public boolean contains(Answer answer) {
+        return this.values.contains(answer);
+    }
+
+    public List<DeleteHistory> delete(User writer) throws CannotDeleteException {
+        boolean isNotOwner = values.stream().anyMatch(answer -> !answer.isOwner(writer));
+        if (isNotOwner) {
+            throw new CannotDeleteException(ExceptionMessage.NO_PERMISSION_DELETE_ANSWER);
+        }
```

Comment:
```suggestion
        // 다른 사람이 쓴 답변이 있어 삭제할 수 없는 검증
        validateDeleted(); // 좀 더 적절한 의미를 가지면 좋을것 같네요 :)
```

해당 코드를 보고 어떠한 의미인지 유추하기 위해 시간이 걸릴것 같아요.
그렇다면, 의도를 들어내는 의미를 가진 메소드로 분리한다면 좀 더 코드가 직관적일 것 같아요!

Reply:
태화님이 말씀주신 것처림 의미있는 메소드로 분리한다면 코드를 이해하는데 더 좋을 것 같습니다!
의견 감사합니다😃